packages/react-storage/src/components/StorageBrowser/Views/Controls/Overwrite.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { CLASS_BASE } from '../constants';
+import {
+  InputElement,
+  ViewElement,
+  LabelElement,
+} from '../../context/elements/definitions';
+
+const BLOCK_NAME = `${CLASS_BASE}__overwrite`;
+export const LABEL_TEXT = "Don't overwrite existing files with the same name.";
+
+interface OverwriteControlProps {
+  checked?: boolean;
+  handleChange?: () => void;
+}
+
+export const OverwriteControl = ({
+  checked,
+  handleChange,
+}: OverwriteControlProps): React.JSX.Element => (
+  <ViewElement className={BLOCK_NAME}>
+    <InputElement
+      checked={checked}
```

Comment:
Curious, do we need to provide `checked` since the `InputElement` will update based on clicks natively?

Reply:
Thanks, yeah we don't need that. I updated checked to defaultChecked in case we need to change the default checked status at some point.